index.d.ts

```diff
@@ -197,13 +197,14 @@ interface LogoutOptions {
 /**
  * Configuration parameters passed to the `auth()` middleware.
  *
- * {@link ConfigParams.issuerBaseURL issuerBaseURL}, {@link ConfigParams.baseURL baseURL}, {@link ConfigParams.clientID clientID}
+ * {@link ConfigParams.issuerBaseURL issuerBaseURL}, {@link ConfigParams.baseURL baseURL}, {@link ConfigParams.clientID clientID}, {@link ConfigParams.clientSecret clientSecret}
```

Comment:
`clientSecret` isn't required by default, only if you're doing code flow

Reply:
True 👍 but it can be included with env vars so I'll re-word a bit.